style.css

```diff
@@ -0,0 +1,78 @@
+*{
+    margin: 0%;
+    padding: 0%;
+    box-sizing: border-box;
+}
+body{
+    display: flex;
+    justify-content: center;
+    align-self: center;
+    min-height: 100vh;
+
+    background: #222;
+}
+.container{
+    position: relative;
+
+    max-width: 100%;
+    margin: 40px;
+
+    margin-left: 20px;
+    margin-right: 20px;
```

Comment:
We can use margin shorthand here eg: margin: 40px 20px; 
40px is for top and bottom & 20px is for left and right